4_select_blog.md

```diff
@@ -0,0 +1,5 @@
+```
+Câu 4:Select 10 blog mới nhất đã active
+
+SELECT * FROM blog WHERE is_active <> 0 ORDER BY id desc limit 10
```

Comment:
Nên tránh toán tử phủ định nếu có thể để tăng performance. thay vì `is_active  <> 0`  thì nên dùng `is_active  = 1`